packages/app/src/app/pages/Sandbox/Editor/index.js

```diff
@@ -132,9 +135,10 @@ class ContentSplit extends React.Component {
               </SplitPane>
 
               {vscode && (
-                <AOverride
+                <StatusBar
                   style={{
                     position: 'fixed',
+                    display: statusBar ? 'block' : 'none',
```

Comment:
Maybe don't render the component if `statusBar` doesn't have a value? 🤔

```ts
{vscode && statusBar && (
  // ...
)}
```

Reply:
That's on porpuse because the sidebar won't remount again if I do that

Reply:
Maybe add a comment to make this more clear then? 🤔
This way people who work on this piece in the future won't do it the way I suggested.